rules/linux/lnx_system_info_discovery.yml

```diff
@@ -44,4 +41,7 @@ detection:
           - '/sys/class/dmi/id/chassis_vendor'
           - '/proc/scsi/scsi'
           - '/proc/ide/hd0/model'
+          - '/proc/version'
+          - '/etc/redhat-release'
```

Comment:
could you please update this section using [this](http://paperlined.org/rosetta_stone/os/info_utils/Linux_OS_version.html) information?

Reply:
thanks!